packages/netskope/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.18.0"
+  changes:
+    - description: Added support for custom tcp options and upgraded stack requirements to kibana ^8.7.0
```

```suggestion
    - description: Added support for custom TCP options.
```
I don't think we need to note stack version changes in the changelog.